chain/vm/gas.go

```diff
@@ -111,6 +111,7 @@ var Prices = map[abi.ChainEpoch]Pricelist{
 		verifySignature: map[crypto.SigType]int64{
 			crypto.SigTypeBLS:       16598605,
 			crypto.SigTypeSecp256k1: 1637292,
+			crypto.SigTypeDelegated: 1637292,
```

Comment:
Don't we also need this in pricelists for other versions?

Reply:
No, nothing should ever call it anyway?